podpac/core/coordinates/coordinates.py

```diff
@@ -879,9 +863,9 @@ def geotransform(self):
         # Do the uniform coordinates case
         if (
             "lat" in self.dims
-            and isinstance(self._coords["lat"], UniformCoordinates1d)
             and "lon" in self.dims
-            and isinstance(self._coords["lon"], UniformCoordinates1d)
+            and self._coords["lat"].is_uniform
+            and self._coords["lon"].is_uniform

```

Comment:
@mpu-creare I added start, step, and stop to ArrayCoordinates1d so now this works for array coordinates, too.